libraries/AP_Motors/AP_MotorsHeli_Single.cpp

```diff
@@ -137,6 +139,50 @@ const AP_Param::GroupInfo AP_MotorsHeli_Single::var_info[] = {
     // @User: Standard
     AP_GROUPINFO("COL2YAW", 21,  AP_MotorsHeli_Single, _collective_yaw_scale, 0),
 
+    // @Param: DDFP_THST_EXPO
+    // @DisplayName: DDFP Tail Rotor Thrust Curve Expo
+    // @Description: Tail rotor DDFP motor thrust curve exponent (0.0 for linear to 1.0 for second order curve)
+    // @Range: -1 1
+    // @User: Standard
+    AP_GROUPINFO("DDFP_THST_EXPO", 22, AP_MotorsHeli_Single,  _ddfp_thst_expo, 0.55),
```

Comment:
better as a subgroup to save indexes, possibly with DDFP_EXPO_EN ?

Reply:
added param table to thrust linearization class and have heli using it as a subgroup.